### PR TITLE
[NEW UI] Radio card refacto

### PR DESCRIPTION
**Pull request by tblivet**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | [NEW UI] Radio card refacto for front integration
| Type?             | refactor
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | -
| Sponsor company   | @PrestaShopCorp
| How to test?      | -

### CODE REVIEW

File: views/templates/components/radio-card.html.twig
Comment (ga-devfront) [inline views/templates/components/radio-card.html.twig]: I think we have to rename this variable because it's the action of check.

File: views/templates/components/radio-card.html.twig
Comment (ga-devfront) [inline views/templates/components/radio-card.html.twig]: can be present inside dom only if we give a list of check

**Review by ga-devfront — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
